### PR TITLE
Enable top instructions and tts

### PR DESCRIPTION
**Pull request by Hamms**
~~TODO: point this PR to staging as the base branch once https://github.com/code-dot-org/code-dot-org/pull/11297 is merged.~~

### CODE REVIEW

**Comment by Hamms** [comment]
You didn't used to be able to, but at some point they changed it; hitting the "Edit" button in the top right allows you up edit the PR's title OR base branch!

File: apps/src/templates/instructions/ChatBubble.jsx [hunk 1]
```diff
@@ -15,7 +14,10 @@ const styles = {
     backgroundColor: 'white',
     borderRadius: 10,
     margin: '5px 0',
-    padding: '5px 10px',
+    paddingTop: 5,
+    paddingBottom: 5,
+    paddingLeft: 10,
+    paddingRight: 10,
```
Comment (Hamms) [inline apps/src/templates/instructions/ChatBubble.jsx:20]: updated to correct a Radium warning about mixing "padding" declaration styles.

**Comment by Hamms** [comment]
@balderdash PTAL; I forgot that we want to only enable TTS for K1

**Comment by Hamms** [comment]
PTAL; I forgot that most of our tests would end up referencing the dialog in some way or another, so I had to modify a bunch of em.

**Comment by joshlory** [comment]
LGTM

**Pull request merged**